src/test/java/jp/co/soramitsu/sora/didresolver/domain/iroha/repositories/AccountRepositoryTest.java

```diff
@@ -0,0 +1,109 @@
+package jp.co.soramitsu.sora.didresolver.domain.iroha.repositories;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace.NONE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Collections;
+import javax.sql.DataSource;
+import jp.co.soramitsu.sora.didresolver.domain.iroha.entities.Account;
+import jp.co.soramitsu.sora.didresolver.domain.iroha.entities.AccountVault;
+import jp.co.soramitsu.sora.didresolver.dto.DDO;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.data.repository.init.Jackson2RepositoryPopulatorFactoryBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+/**
+ * TODO: resolve why can't properly shut down EntityManagerFactory (probably resource closing
```

Comment:
please provide a case when this happens

Reply:
Whenever gradle test is ran. Possibly not worth resolving